authentication/securestore/main.go

```diff
@@ -207,6 +210,55 @@ func ValidateBackend(backend string) error {
 	return nil
 }
 
+func keychainGet() ([]byte, error) {
+	credentials, err := gokeyring.Get(ItemKey, ItemKey)
+	if err != nil && err != gokeyring.ErrNotFound {
+		return []byte(credentials), err
+	} else if err == gokeyring.ErrNotFound {
+		return []byte(""), nil
+	}
```

Comment:
The linter is suggesting using err.Is due to `Comparison with errors using equality operators fails on wrapped errors`